content/tutorials/r_setup_codingclub_session/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Tutorial setup_codingclub_session()"
+description: "Tutorial on how to use setup_codingclub_session() to load data and scripts to use during coding clubs."
+author: "Emma Cartuyvels"
+date: 2020-10-26
+categories: ["r"]
+tags: ["data", "r"]
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(eval = FALSE)
+```
+
+
+Before each coding club you will usually have to download some data and some scripts. This can be done manually by going to each of the GitHub pages but it is much easier to use the setup_codingclub_session() function of the inborutils package.
```

Comment:
```suggestion
Before each coding club you will usually have to download some data and some scripts.
This can be done manually by going to each of the GitHub pages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
```